WordPress/Classes/Services/CommentService.m

```diff
@@ -1141,7 +1142,7 @@ - (Comment *)findCommentWithID:(NSNumber *)commentID fromPost:(ReaderPost *)post
 
 - (void)updateComment:(Comment *)comment withRemoteComment:(RemoteComment *)remoteComment
 {
-    comment.commentID = [remoteComment.commentID intValue] ?: 0;
```

Comment:
I'm a little leery about removing the defaults here.  I know core data _should_ use the default that is declared in the model editor, but RemoteContent still lacks nullibility modifiers and can technically still pass nil as a value.  I'm not quite sure how this would be handled when a property is defined as optional but has a default and we might want to keep these just to be safe.  Thoughts?

Reply:
I did test with `RemoteComment`s not having those values at all, and it works fine. So I felt comfortable removing them. But if you'd still prefer, I can add them back.

Reply:
Nah! I'm being paranoid and our tests and conversation in Slack demonstrate this is fine.  Thanks for talking (and testing) through it!